### PR TITLE
Review SMTP authentication section

### PR DESCRIPTION
**Pull request by javimed**
## Description
This PR does the following:

- Adds a configuration line to the SMTP server configuration file.
- Instructs the reader on what option to select while installing for the mail server configuration.
- Reviews some texts and steps.

## Checks
- [X] Compiles without warnings.
- [X] Uses present tense, active voice, and semi-formal registry.
- [X] Uses short, simple sentences.
- [X] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [X] Uses three spaces indentation.
- [X] Adds or updates meta descriptions accordingly.
- [X] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/user-manual/manager/manual-email-report/smtp-authentication.rst [hunk 1]
```diff
@@ -1,56 +1,57 @@
...
+   :description: Learn how to use Postfix to send email alerts using Gmail.
 
 SMTP server with authentication
 ===============================
 
-If your SMTP server uses authentication (like Gmail, for instance), a server relay will need to be configured as Wazuh does not support this. **Postfix** can be configured to provide this capability. The following guide describes the minimal configuration needed to use Postfix to send emails:
+Wazuh email alerts does not support SMTP servers with authentication such as Gmail. However, you can use a server relay, like Postfix, to send these emails. Follow this guide for instructions on configuring Postfix with Gmail.
 
-#. Install the needed packages:
+#. Run this command to install the required packages. Select **No configuration** when prompted about the **Mail server configuration type**.
```
Comment (s-ocando) [inline source/user-manual/manager/manual-email-report/smtp-authentication.rst:11]:
```suggestion
#. Run this command to install the required packages. Select **No configuration** if prompted about the **Mail server configuration type**.
```

**Review by s-ocando — COMMENTED** [review]
Hi @javimed, 

Please review the suggestion below.

File: source/user-manual/manager/manual-email-report/smtp-authentication.rst [hunk 2]
```diff
@@ -1,56 +1,57 @@
...
 
-      .. group-tab:: APT
+      .. group-tab:: Ubuntu
 
          .. code-block:: console
 
-            # apt-get install postfix mailutils libsasl2-2 ca-certificates libsasl2-modules
+            # apt-get update && apt-get install postfix mailutils libsasl2-2 ca-certificates libsasl2-modules
+
+#. Add these lines to the end of the ``/etc/postfix/main.cf`` file to configure Postfix.
```
[the remark this answers is not in the document]
Comment (javimed) [reply source/user-manual/manager/manual-email-report/smtp-authentication.rst:27]: True. It's necessary to create it and "add these lines" is confusing then

File: source/user-manual/manager/manual-email-report/smtp-authentication.rst (hunk 1 above)
Comment (s-ocando) [reply source/user-manual/manager/manual-email-report/smtp-authentication.rst:11]: On CentOS7, for example, there's no prompt to select the configuration type.

**Review by s-ocando — APPROVED** [review]

**Pull request merged**